src/main/java/org/opentripplanner/middleware/auth/Auth0Connection.java

```diff
@@ -67,7 +68,7 @@ public static void checkUser(Request req) {
             DecodedJWT jwt = verifier.verify(token);
             Auth0UserProfile profile = new Auth0UserProfile(jwt);
             if (!isValidUser(profile)) {
-                if (isCreatingSelf(req, profile)) {
+                if (isCreatingOrEmailingSelf(req, profile)) {
```

Comment:
It might be good to rename this to something a bit more generic: `missingProfileIsExpected(req, profile)`. And the below log statement could be generalized a bit too.